hygen/lib/properties/properties.js

```diff
@@ -0,0 +1,22 @@
+const getApiDocs = require("../api/api-spec");
+const tab = "  ";
+
+/**
+ *
+ * @param {string} url
+ * @param {string} args
+ * @returns {Array<string>}
+ */
+module.exports = function (url, args) {
+  const properties = getApiDocs(url, args);
+  const lines = [];
+
+  properties.forEach((property) => {
+    if (Array.isArray(property)) {
+      property.pop();
+      lines.push(tab.repeat(4) + "'" + property + "',");
```

Comment:
are you concatenating an array and a string? I don't understand

Reply:
This is what happens if you call just the array as an string 
![image](https://user-images.githubusercontent.com/101095813/176670666-dad9d8ef-cd3c-4192-b852-4cd3013d9a46.png)